packages/@o3r/configuration/schemas/configuration.metadata.schema.json

```diff
@@ -186,6 +186,13 @@
               "items": {
                 "type": "string"
               }
+            },
+            "restrictionKeys": {
+              "type": "array",
```

Comment:
```suggestion
              "type": "array",
              "minItems": 1,
```

Reply:
To be discussed but for me it is optional to provide `restrictionKeys` so we can either not provide it, either provide an empty array, both are correct.
Maybe someone from @AmadeusITGroup/otter_cms can confirm ?